arkouda-env-dev.yml

```diff
@@ -1,20 +1,20 @@
 name: arkouda-env-dev
 channels:
   - conda-forge
+  - defaults
 dependencies:
   - python>=3.7   # minimum 3.7
   - numpy>=1.16.5,<=1.21.5
   - pandas>=1.1.0
   - pyzmq>=20.0.0
-  - typeguard==2.10.0
   - tabulate
   - pyfiglet
   - versioneer
   - matplotlib
   - h5py
   - pip
   - types-tabulate
-  - tables
+  - pytables
   - pyarrow
 
   # Developer dependencies

```

Comment:
do we want to add any other developer dependencies? Ones that come to mind are `jupyter` (which will bring along ipython) and the formatting ones (`black`, `flake8`, `isort`)

Reply:
`Jupyter` I would say no because Arkouda does not require. The formatting stuff could be added. However, it looks like you have an issue in for documenting the formatting (#1395). I would add it with the documentation. This modification was mainly to resolve issues related to the parquet dependency.